### PR TITLE
Efficiency: Compile RegEx once

### PR DESCRIPTION
**Pull request by xErik**
### Description

Efficiency: Create RegEx-object only once.

### CODE REVIEW

File: src/js/extensions/auto-link.js [hunk 1]
```diff
@@ -207,15 +210,14 @@
...
-                textContent = contenteditable.textContent,
+            var textContent = contenteditable.textContent,
                 match = null,
                 matches = [];
 
-            while ((match = linkRegExp.exec(textContent)) !== null) {
+            while ((match = LINK_REGEXP.exec(textContent)) !== null) {
                 var matchOk = true,
                     matchEnd = match.index + match[0].length;
-                // If the regexp detected something as a link that has text immediately preceding/following it, bail out.
```
Comment (j0k3r) [inline src/js/extensions/auto-link.js]: Could you revert that change?

Comment (xErik) [inline src/js/extensions/auto-link.js]: Probably, just a sec.

**Comment by xErik** [comment]
Err, I am not a Github-Wizard. Is this the comment there?

**Comment by j0k3r** [comment]
@xErik yep it's ok ! Thanks

**Pull request merged**